src/api/spec/features/webui/groups_spec.rb

```diff
@@ -58,7 +58,13 @@ def group_in_datatable(page, group)
     visit group_path(group_1)
 
     within(find('div.group-user', text: admin.login)) do
-      click_link('Remove member from group')
+      # HACK: Click on a button that has zero size. The delete button is a
```

Comment:
I had to add this because I was getting failures on the feature tests regarding the delete button not being clickable:
```
Selenium::WebDriver::Error::ElementNotInteractableError:                                                                        
       element not interactable: element has zero size
```
The problem was that the button element is a floating one, and the Capybara driver cannot click on it.
I'm not 100% about using this hack, @ncounter can we somehow change the delete button so it's not a floating element?

Reply:
Cool, so this will be changed? It sounds like it should.

Reply:
I'm fixing it now